test/core.js

```diff
@@ -47,6 +47,49 @@ test( "jQuery(html) loose rules", function() {
 	}
 });
 
+test( "selector state", function() {
+	expect( 18 );
+
+	var test;
+
+	test = jQuery( undefined );
+	equal( test.selector, "", "Empty jQuery Selector" );
+	equal( test.context, undefined, "Empty jQuery Context" );
+
+	test = jQuery( document );
+	equal( test.selector, "", "Document Selector" );
```

Comment:
OK, I just wanted to get it tested properly. Since you checked it on your own, I'm good. :)